examples/dac/dac_main.c

```diff
@@ -110,15 +110,15 @@ static void print_cmds(FAR const char *header,
                        size_t ncmds,
                        FAR const char *trailer)
 {
-  printf(header);
+  printf("%s", header);
```

Comment:
Sorry, but what was the warning here? The `header` seems to be `const char*`

Reply:
Here:
```
Error: format string is not a string literal (potentially insecure)
```
In general is dangerous to printf(str) and the content of str can't be determined in compile time since str may contain % and trigger printf do some bad action.

Reply:
The printf is used below, so let's go with the current change. Otherwise I would recommend to use puts